mail/templates/verification/body.html

```diff
@@ -34,9 +34,13 @@ <h1 style="margin: 0 0 10px; font-size: 25px; line-height: 30px; color: #03152d;
               <tr>
                   <td style="padding: 20px; font-family: sans-serif; font-size: 15px; line-height: 20px; color: #555555;">
                       <p style="margin: 0 0 10px;">
-                        Welcome and thanks!
+                        Welcome and thank you!
                         <br/>
-                        The {{ site_name }} Team
+                        The {{ site_name }} Admissions Team
+                        <br/>
+                        MIT Bootcamps, 77 Massachusetts Avenue, Cambridge, MA 02139
+                        <br/>
+                        © 2020 MIT Bootcamps, All Rights Reserved. Terms of Service and Privacy Policy
```

Comment:
Should Terms of Service and Privacy Policy be links?

Reply:
👍